docs/source/GetStartedGuide.rst

```diff
@@ -138,6 +138,12 @@ expectations, it passed! These are known as `simple tests`, but there is also an
 type of test, which we call `instrumented tests`. See more at :ref:`test-types` or just
 keep reading.
 
+.. note:: Although in most cases running ``avocado run $test1 $test3 ...` is
+          fine, it can lead to argument vs. test name clashes. The safest
+          way to execute tests is ``avocado run --$argument1 --$argument2
+          -- $test1 $test2``. Everything after `--` will be considered
+          positional argument, therefor test name (in case of ``avocado run``)
```

Comment:
* typo: therefor
* I think you should use the plural for argument and test name.